README.md

```diff
@@ -35,7 +35,7 @@ steps:
 - uses: actions/checkout@master
 - uses: codecov/codecov-action@v3
   with:
-    token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
```

Comment:
Is this a breaking change? Is it now required for public repos?

Reply:
@3flex this is not a breaking change, but we do suggest using the token when possible to prevent flakiness when hitting the GitHub API